Form-Controls/index.html

```diff
@@ -13,10 +13,124 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <!DOCTYPE html>
```

Comment:
Your html structure is wrong here. Please, update it. You are in the middle of a html doc, but `<!DOCTYPE html>` is always the very first element you see in the html.